jobs/scheduled/enqueue_reminders.rb

```diff
@@ -12,26 +12,39 @@ def execute(_args)
     private
 
     def skip_enqueue?
-      SiteSetting.remind_assigns_frequency.nil? || SiteSetting.remind_assigns_frequency.zero?
+      SiteSetting.remind_assigns_frequency.nil? || !SiteSetting.assign_enabled?
     end
 
     def user_ids
-      interval = SiteSetting.remind_assigns_frequency
+      global_frequency = SiteSetting.remind_assigns_frequency.to_i
```

Comment:
Do we need the `.to_i` here? My understanding is that `SiteSetting` will cast the value automatically when it retrieves from the DB.